module-os/application.c

```diff
@@ -86,7 +84,4 @@ void vApplicationStackOverflowHook(TaskHandle_t xTask, signed char *pcTaskName)
 #endif
 
 void vApplicationTickHook()
-{
-    //    static uint8_t status = 0;
-    //    GPIO_PinWrite(base, pin, output)
-}
+{}
```

Comment:
is it in use? maybe remove?

Reply:
This is enabled by the specific configuration option inside `FreeRTOSConfig.h`. Sure we can remove it if it's not used.